CMakeLists.txt

```diff
@@ -319,6 +334,14 @@ add_executable(lovr
   src/lib/vec/vec.c
 )
 
+if (ANDROID)
+  # For Android only we build as a library.
```

Comment:
I would prefer a more semantic condition like "build as (shared) library" rather than "on android we happen to build as a lib".

Reply:
on Android it's impossible to build as anything other than a shared library. there's no concept of an executable. I'l switch this to LOVR_BUILD_SHARED and have this set to ON when if (ANDROID)